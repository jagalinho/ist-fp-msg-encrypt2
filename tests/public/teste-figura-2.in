letras = ('A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z')
mgc =  "MENSAGEM GERACAO DE CHAVE"
chave = ()
chave = gera_chave_linhas(letras, mgc)
fig = figura('IZ', chave)
fig
