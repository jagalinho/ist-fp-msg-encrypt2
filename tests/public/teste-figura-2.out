('c', faz_pos(2, 4),  faz_pos(4, 4))
